### PR TITLE
Maintenance loot cleanup

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
fix: All tophats, rather than just the ones in maintenance, hurt a tiny
bit if you throw them at people.
/:cl:

- All tophats have throwforce 1
- Cable coils are now random colours
- Cable coils now actually come in stacks of 5 in maint

### CODE REVIEW

File: code/game/objects/items/stacks/rods.dm [hunk 1]
```diff
@@ -73,3 +73,12 @@ var/global/list/datum/stack_recipe/rod_recipes = list ( \
 
 /obj/item/stack/rods/cyborg/update_icon()
 	return
+
+/obj/item/stack/rods/ten
+	amount = 10
+
+/obj/item/stack/rods/twentythree
```
Comment (RemieRichards) [inline code/game/objects/items/stacks/rods.dm]: I prefer modified types to something as nonsensical as a ```/twentythree``` type

Comment (coiax) [inline code/game/objects/items/stacks/rods.dm]: I don't know why it's 23 rods anyway.

**Comment by ExcessiveUseOfCobblestone** [comment]
img

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -21,6 +21,7 @@
 	icon_state = "tophat"
 	item_state = "that"
 	dog_fashion = /datum/dog_fashion/head
+	throwforce = 1
```
Comment (Iamgoofball) [inline code/modules/clothing/head/misc.dm]: :-1: buff to 10

**Comment by octareenroon91** [comment]
5 units of coil seems kind of harsh

File: code/modules/clothing/head/misc.dm (hunk 2 above)
Comment (Supermichael777) [inline code/modules/clothing/head/misc.dm]: buff to 1000 against statues and zero against humans

**Comment by PKPenguin321** [comment]
Yeah, what's with the coil change

**Comment by coiax** [comment]
There is no coil change.

**Comment by octareenroon91** [comment]
Now I see

File: code/game/objects/items/stacks/rods.dm (hunk 1 above)
Comment (octareenroon91) [inline code/game/objects/items/stacks/rods.dm]: May as well randomize about 23 if you're going for an odd number.

File: code/modules/clothing/head/misc.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/head/misc.dm]: make the user take double electricity damage if they touch it while on a powered wire.

File: code/game/objects/items/stacks/rods.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/objects/items/stacks/rods.dm]: coix psl

Comment (coiax) [inline code/game/objects/items/stacks/rods.dm]: how about I just change it to 25

Comment (optimumtact) [inline code/game/objects/items/stacks/rods.dm]: This crusade against modified types is silly, in this case it's perfectly fine to adjust the amount var of the stacks, the amount var is very unlikely to go away at any point.

Comment (coiax) [inline code/game/objects/items/stacks/rods.dm]: but when does the modification happen? What if the type needs to update_icon?

Comment (RemieRichards) [inline code/game/objects/items/stacks/rods.dm]: it does exactly what you've done here, it's a compile time creation of a new type.
that's it, think of them as anonymous types/classes, they're defined and used once (unless stored in a var)

Comment (coiax) [inline code/game/objects/items/stacks/rods.dm]: Removing the modified types will turn this into a hat balance PR, but okay

Comment (optimumtact) [inline code/game/objects/items/stacks/rods.dm]: I don't care about the hat balance one, that's completely fine, it's this stack one I object to the most.

File: code/modules/clothing/head/misc.dm (hunk 2 above)
Comment (Tacolizard) [inline code/modules/clothing/head/misc.dm]: @ExcessiveUseOfCobblestone why

**Comment by coiax** [comment]
Done.

**Pull request merged**